x-pack/test/security_solution_cypress/runner.ts

```diff
@@ -5,15 +5,33 @@
  * 2.0.
  */
 
+import { chunk } from 'lodash';
 import { resolve } from 'path';
+import glob from 'glob';
+
 import Url from 'url';
 
 import { withProcRunner } from '@kbn/dev-proc-runner';
 
 import semver from 'semver';
 import { FtrProviderContext } from './ftr_provider_context';
 
-export async function SecuritySolutionCypressCliTestRunner({ getService }: FtrProviderContext) {
+const retrieveIntegrations = (chunksTotal: number, chunkIndex: number) => {
+  const pattern = resolve(
+    __dirname,
+    '../../plugins/security_solution/cypress/integration/**/*.spec.ts'
+  );
+  const integrationsPaths = glob.sync(pattern);
+  const chunkSize = Math.ceil(integrationsPaths.length / chunksTotal);
+
+  return chunk(integrationsPaths, chunkSize)[chunkIndex - 1];
+};
+
+export async function SecuritySolutionConfigurableCypressCliTestRunner(
```

Comment:
Lots of thanks for this change!! I really like the way you have refactored it!! :D